.github/workflows/general.yml

```diff
@@ -55,6 +55,20 @@ jobs:
           run: cargo test --release -- --include-ignored
         - name: Run fuzzer
           run: cargo run --release --bin=fuzzer
+  check_nix:
+    name: Check nix code
+    strategy: 
+      matrix: 
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Helpful idea to iterate on operating systems in the CI! I think we could do that for the fuzzer as well (which is independent of this PR, but just pointing it out to ourselves). Do the different systems run in parallel?

Reply:
yes, they do, it generates a matrix that will run on different runners in parallel :)

Reply:
Nice! We'll follow-up on this in #153.